app/public/fullcalendar-5.5.1/LICENSE.txt

```diff
@@ -0,0 +1,22 @@
+MIT License
```

Comment:
Can you remove this folder of fullcalendar-5.5.1 in the public directory? bc this cause you to have 113 file changes.